samples/Sentry.Samples.Maui/MauiProgram.cs

```diff
@@ -22,8 +22,13 @@ public static MauiApp CreateMauiApp()
 
                 options.Debug = true;
                 options.SampleRate = 1.0F;
+#if ANDROID
+                options.Native.ExperimentalOptions.SessionReplay.OnErrorSampleRate = 1.0;
+                options.Native.ExperimentalOptions.SessionReplay.SessionSampleRate = 1.0;
+                options.Native.ExperimentalOptions.SessionReplay.RedactAllImages = false;
+                options.Native.ExperimentalOptions.SessionReplay.RedactAllText = false;
+#endif
```

Comment:
In the future, we should probably have a facade that configures options for the current platform (Android/iOS), instead of letting the user specify them multiple times.

Reply:
> In the future, we should probably have a facade that configures options for the current platform (Android/iOS), instead of letting the user specify them multiple times.

If the options had the same name for both iOS and Android (`OnErrorSampleRate`, `SessionSampleRate` etc.) then presumably SDK users could write something like this, even without any facade:

```
#if ANDROID || IOS
                options.Native.ExperimentalOptions.SessionReplay.OnErrorSampleRate = 1.0;
                options.Native.ExperimentalOptions.SessionReplay.SessionSampleRate = 1.0;
                options.Native.ExperimentalOptions.SessionReplay.RedactAllImages = false;
                options.Native.ExperimentalOptions.SessionReplay.RedactAllText = false;
#endif
```

When targeting android, `options.Native` would resolve to the appropriate platform specific options at build no?